### ISSUE TITLE
Electron applications does not read if aria-atomic is false in JAWS

### ISSUE THREAD

**Issue by Tanuja-Kulal** [electron/electron#13299]
* Electron Version: 2.0.0
* Operating System (Platform and Version): Windows 10 Enterprise 10.0.17134 Build 17134
* Last known working Electron version:  not sure

**Summary**
I work on electron application which renders in chrome. Jaws does not read text in live region unless aria-atomic = true is mentioned. There are few logs which I want to read with aria-live="polite" and aria-atomic="false" settings. In this scenario JAWS does not read live region but NVDA does. I logged bug against JAWS ( https://github.com/FreedomScientific/VFO-standards-support/issues/89) and they pointed out it to be Electron bug.

**Repro steps**
1. clone https://github.com/Tanuja-Kulal/ElectronApp
2. follow instructions in ReadMe.md
3. Run electron application
4. click on "display random number" button

**Expected result**
The new numbers should read out

**Actual result**
The numbers are not read.

**Additional Information**
Toggle aria-atomic flag in index.html and try. JAWS reading live changes..
JAWS version and build number: 2018.1805.33 ILM
Logged bug against JAWS: https://github.com/FreedomScientific/VFO-standards-support/issues/89

**Comment by Tanuja-Kulal** [comment]
Is there any update on this?

**Comment by sofianguy** [comment]
Thank you for taking the time to report this issue and helping to make Electron better.

The version of Electron you reported this on has been superseded by newer releases. 

If you're still experiencing this issue in Electron v4.2.x or later, please add a comment specifying the version you're testing with and any other new information that a maintainer trying to reproduce the issue should know.

I'm setting the `blocked/need-info` label for the above reasons. This issue will be closed 7 days from now if there is no response.

Thanks in advance! Your help is appreciated.

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
